packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -317,7 +316,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     const {
       text,
       children,
-      secondaryText = this.props.description,
+      secondaryText = this.props.secondaryText,
```

Comment:
I think this was a missed deprecation reversion

Reply:
Fixed in latest commit.